frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -114,6 +143,13 @@ class DatasetUploadView extends React.PureComponent<PropsWithForm, State> {
                   <Upload.Dragger
                     name="files"
                     beforeUpload={file => {
+                      if (!form.getFieldValue("name") || form.getFieldValue("name") === "") {
+                        const filename = file.name
+                          .split(".")
+                          .slice(0, -1)
+                          .join(".");
+                        form.setFieldsValue({ name: filename });
+                      }
```

Comment:
@grittaweisheit Wanted the file name to be the default dataset name if no name was entered before. I just added this here as I think this is a very good idea